### DISCUSSION TITLE
Special output for `:histogram` format images

### DISCUSSION THREAD

**Discussion by kipcole9** [akash-akya/vix#156] [Ideas]
As I work more with LUTs, curves and image mapping the more I realise it would be helpful to have those curves presented differently from normal images.  My idea - which I am happy to implement - is to have the Kino output for a histogram image be rendered as a Vegalite chart, with a second tab showing the min, max, mean and mode of the histogram.

If you’re ok with that idea, I’ll work up a PR over the next few days.

**Comment by akash-akya** [comment]
Hi @kipcole9, that sounds like a good idea, I assume you will use `vips_interpretation` as a hint to decide if the image is histogram or not. Please go ahead with the implementation at your convenience 

I think we can use similar logic to show matrix & fourier transformation images, I will add those afterwards.

Thanks!
